js/packages/cli/src/candy-machine-v1-cli.ts

```diff
@@ -362,12 +351,13 @@ programCommand('verify_token_metadata')
     log.info(`Elapsed time: ${timeTaken}\n`);
   });
 
-programCommand('verify')
+programCommand('verify_upload')
   .option(
     '-r, --rpc-url <string>',
     'custom rpc url since this is a heavy command',
   )
   .action(async (directory, cmd) => {
```

Comment:
Are these being used?

Reply:
Good question. I don't think this command can be used any more since the deprecation.

Reply:
The upload command can still be used to modify the config account. This would just verify that it modifies it correctly

Reply:
Perhaps altering the command name to something like `verify_config_account` or something like that would be more clear?

Reply:
It's probably fine for now. I'm not going to dig too deep into V1 stuff. I assume we're just going to out right delete these sometime soon.